prefect_docker/containers.py

```diff
@@ -0,0 +1,63 @@
+"""Integrations with Docker Containers."""
+
+from typing import Any, Container, Dict, List, Optional, Union
+
+from prefect import get_run_logger, task
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+
+from prefect_docker.host import DockerHost
+
+
+@task
+async def create_docker_container(
+    image: str,
+    command: Optional[Union[str, List[str]]] = None,
+    name: Optional[str] = None,
+    detach: Optional[bool] = None,
+    docker_host: Optional[DockerHost] = None,
+    **create_kwargs: Dict[str, Any],
+) -> Container:
+    """
+    Create a container without starting it. Similar to docker create.
+
+    Args:
+        image: The image to run.
+        command: The command(s) to run in the container.
+        name: The name for this container.
+        detach: Run container in the background.
+        docker_host: Settings for interacting with a Docker host.
+        **create_kwargs: Additional keyword arguments to pass to
+            [`client.containers.create`](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.ContainerCollection.create).
+
+    Returns:
+        A Docker Container object.
+
+    Examples:
```

Comment:
You could also potentially include an example where a custom Docker host is used. Not strictly necessary, but it would show the flexibility of the task.

Reply:
I added it to README since it can apply to other tasks as well.